src/App/App.js

```diff
@@ -44,6 +48,34 @@ function App({ dexieCurrentUserInstance }) {
     [logoutMermaid],
   )
 
+  const handleSyncPushErrors = (projectsWithSyncErrors) => {
+    // projectsWithSyncErrors's type: { projectId: { name: string, apiDataTablesThatRejectedSyncing: string[] } }
```

Comment:
k, seriously next project we work together on from scratch, lets use TypeScript!

Reply:
I never need convincing to use Typescript, big fan

Reply:
omg omg omg omg